CMPUT_301_Project/app/src/main/res/layout/fragment_sign_up.xml

```diff
@@ -0,0 +1,161 @@
+                    android:layout_weight="1"
+                    android:text="Email:"
+                    android:textSize="24sp" />
+
+                <EditText
+                    android:id="@+id/createEmailEV"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="2"
+                    android:ems="10"
+                    android:inputType="textPersonName" />
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginTop="32dp"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/createUsernameTV"
+                    android:layout_width="wrap_content"
+                    android:layout_height="fill_parent"
+                    android:layout_weight="1"
+                    android:text="Username:"
+                    android:textSize="24sp" />
+
+                <EditText
+                    android:id="@+id/createUsernameEV"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="4"
+                    android:ems="10"
+                    android:inputType="textPersonName" />
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginTop="32dp"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/createPasswordTV"
+                    android:layout_width="wrap_content"
+                    android:layout_height="fill_parent"
+                    android:layout_weight="1"
+                    android:text="Password:"
+                    android:textSize="24sp" />
+
+                <EditText
+                    android:id="@+id/createPasswordEV"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="4"
+                    android:ems="10"
+                    android:inputType="textPassword" />
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginTop="32dp"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/reenterPasswordTV"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:lines="2"
+                    android:text="Re-Enter Password:"
```

Comment:
Can we put this in a layout and stack it over two lines? It looks very small at the moment.

Reply:
I agree, the text on "Re-Enter Password:" looks small

Reply:
Problem was with layout widths not working with chosen text size for textviews. Changed all sizes of textviews to match at a slightly smaller size and now the layouts all lineup. 're-enter password' now spans 2 lines and this should be resolved.